show_notes/Episode_9_Part1.md

```diff
@@ -0,0 +1,42 @@
+# Episode 9, Part 1: Kotlin Gradle plugin, compilers and build systems with Alexey Tsvetkov
+
+**Important note for Pocket Casts app users:**
+Unfortunately Pocket Casts android app has a problem with downloading our episodes (to listen to them offline).
+It seems that while downloading their android app is using a old http client which doesn't support latest TLS cipher standards we use on server side to host our episodes.  
+
+Nevertheless, **streaming our episodes (instead of downloading) works** in Pocket Casts android app.
+
+See this tutorial: [Pocket Casts: Streaming](http://support.pocketcasts.com/article/streaming-episodes/)
+
+
+### [How to listen & subscribe](https://github.com/artem-zinnatullin/TheContext-Podcast)
+
+ - [Discussion after the episode](https://github.com/artem-zinnatullin/TheContext-Podcast/issues/62)
+
+
+In this episode we have talked with Alexey Tsvetkov who works at the Kotlin team at jetbrains about Kotlin's gradle plugin, compilers and build systems in general.
```

Comment:
oops,  no clue what I was thinking about ...
should I change this or what are you trying to say with "(will do)"?